frontend/index.html

```diff
@@ -0,0 +1,65 @@
+<!--
```

Comment:
Why are we adding this file?, we have this within public/ folder

Reply:
spurious file add, not sure how it got in here.